### PR TITLE
Landing page design

### PR DESCRIPTION
**Pull request by karapeoples**
Trying to fix VSCode Folder that was pushed. Landing Page Design Changes.

### CODE REVIEW

**Comment by jonathan-portillo** [comment]
Looks good!!

**Review by jonathan-portillo — DISMISSED** [review]
Looks great keep up the good work!

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,7 @@
 /build
 
 # misc
+.vscode
```
Comment (Brodt7258) [inline .gitignore]: I think you need to add a `/` in front of this, so that it knows it's a folder, not a file.

Comment (karapeoples) [reply .gitignore]: alright making the change and pushing momentarily

**Review by jonathan-portillo — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Looks goods, kara

**Review by Brodt7258 — APPROVED** [review]

**Pull request merged**